CONTRIBUTING.md

```diff
@@ -10,7 +10,21 @@ We really want Parse to be yours, to see it grow and thrive in the open source c
 * Run the tests for the file you are working on with `npm test spec/MyFile.spec.js`
 * Run the tests for the whole project and look at the coverage report to make sure your tests are exhaustive by running `npm test` and looking at (project-root)/lcov-report/parse-server/FileUnderTest.js.html
 * Lint your code by running `npm run lint` to make sure all your code is not gonna be rejected by the CI.
+* Never publish the lib folder.
+
+##### Run your tests against Postgres (optional)
+
+If your pull request introduces a change that may affect the storage or retrieval of objects, you may want to make sure it plays nice with Postgres.
+
+* Run the tests against the postgres database with `PARSE_SERVER_TEST_DB=postgres npm test`. You'll need to have postgres running on your machine an setup [appropriatedly](https://github.com/parse-community/parse-server/blob/master/.travis.yml#L37)
+* If your feature is a mongoDB only feature and the test don't apply for a Postgres database, you can disable them with: 
```

Comment:
`If your feature is a mongoDB only feature and the test don't apply for a Postgres database, you can disable them with:`

- incorrect English all the way.